cfme/configure/configuration/analysis_profile.py

```diff
@@ -0,0 +1,311 @@
+            'name': self.name,
+            'description': self.description,
+            'file': {'table': getattr(self, 'files')},  # TODO table doesn't exist yet
+            'event_log': {'table': getattr(self, 'events')}}  # TODO table doesn't exist yet
+        if self.profile_type == 'Vm':
+            # two more tabs for VM
+            form_values.update({
+                'category': {'category_selection': self.categories},
+                'registry': {'table': self.registry}})  # TODO table doesn't exist yet
+
+        view = navigate_to(self, 'Add')
+        view.form.fill(form_values)
+
+        if cancel:
+            view.cancel.click()
+        else:
+            view.add.click()
+
+        view = self.browser.create_view(AnalysisProfileAllView)
+        assert view.is_displayed
+        view.flash.assert_no_error()
+        view.flash.assert_success_message('Add of new Analysis Profile was cancelled by the user'
+                                          if cancel
+                                          else 'Analysis Profile "{}" was saved'.format(self.name))
+
+    def update(self, updates=None):
+        # TODO
+        # navigate_to(self, 'Edit')
+        # if updates is None:
+        #     fill(self.form, self, action=form_buttons.save)
+        # else:
+        #     fill(self.form, updates, action=form_buttons.save)
+        # flash.assert_no_errors()
+        pass
+
+    def delete(self, cancel=False):
+        """Delete self via details page"""
+        view = navigate_to(self, 'Details')
+        view.toolbar.configuration.item_select("Delete this Analysis Profile",
+                                               handle_alert=not cancel)
+        view = self.create_view(
+            AnalysisProfileDetailsView if cancel else AnalysisProfileAllView)
+        view.flush_widget_cache()
+        assert view.is_displayed
+        if not cancel:
+            view.flash.assert_success_message('Analysis Profile "{}": Delete successful'
+                                              .format(self.description))
+        else:
+            assert view.flash.messages == []
+
+    def copy(self, name=None):
+        # TODO
+        # if not name:
+        #     name = self.name + "copy"
+        # navigate_to(self, 'Copy')
+        # new_profile = AnalysisProfile(name=name, description=self.description,
+        #                               profile_type=self.profile_type, files=self.files)
+        # fill(self.form, {'name': new_profile.name},
+        #      action=form_buttons.add)
+        # flash.assert_success_message('Analysis Profile "{}" was saved'.format(new_profile.name))
+        # return new_profile
+        pass
+
+    @property
+    def exists(self):
+        try:
+            navigate_to(self, 'Details')
+        except (NavigationDestinationNotFound, CandidateNotFound):
+            return False
+        else:
+            return True
+
+    def __str__(self):
+        return self.name
```

Comment:
You should probably also add Fillable mixin and implement as_fill_value with the same functionality, se we can then remove the `__str__` once we get rid of old style models.